_docs-2/administration/upgrading.md

```diff
@@ -4,6 +4,33 @@ category: administration
 order: 7
 ---
 
+## Upgrading from 2.0.x to 2.1
+
+The recommended way to upgrade from a prior 2.0.x release is to stop Accumulo, upgrade to 2.1 and
+then start 2.1. To upgrade from a release prior to 2.0, follow the [below steps](#upgrading-from-189-to-20)
+to upgrade to 2.0 and then perform the upgrade to 2.1. A direct upgrade from releases prior to 2.0
+has not been tested.
```

Comment:
We need to support upgrade from 1.10, since 1.10 is an LTM release and that is a supported upgrade path. So, upgrading from 1.10 should definitely be tested before 2.1 is released.

Reply:
Yeah, that makes sense. I'll update the comment.